types/mojang-minecraft-ui/index.d.ts

```diff
@@ -0,0 +1,271 @@
+ *   .button("April")
+ *   .button("May");
+ *
+ * form.show(players[0]).then((response) => {
+ *   if (response.selection === 3) {
+ *     dimension.runCommand("say I like April too!");
+ *   }
+ * });
+ *
+ * ```
+ *
+ * Manifest Details
+ * ```json
+ * {
+ *   // mojang-minecraft-ui
+ *   "uuid": "2bd50a27-ab5f-4f40-a596-3641627c635e",
+ *   "version": [ 0, 1, 0 ]
+ * }
+ * ```
+ *
+ */
+import * as mojangminecraft from 'mojang-minecraft';
+/**
+ * Builds a simple player form with buttons that let the player
+ * take action.
+ */
+export class ActionFormData {
+    /**
+     * @remarks
+     * Method that sets the body text for the modal form.
+     * @param bodyText
+     */
+    body(bodyText: string): ActionFormData;
+    /**
+     * @remarks
+     * Adds a button to this form with an icon from a resource
+     * pack.
+     * @param text
+     * @param iconPath
+     */
+    button(text: string, iconPath?: string): ActionFormData;
+    /**
+     * @remarks
+     * Creates a new modal form builder.
+     */
+    constructor();
+    /**
+     * @remarks
+     * Creates and shows this modal popup form. Returns
+     * asynchronously when the player confirms or cancels the
+     * dialog.
+     * @param player
+     * Player to show this dialog to.
+     * @throws This function can throw errors.
+     */
+    show(player: mojangminecraft.Player): Promise<ActionFormResponse>;
+    /**
+     * @remarks
+     * This builder method sets the title for the modal dialog.
+     * @param titleText
+     */
+    title(titleText: string): ActionFormData;
+}
+/**
+ * Returns data about the player results from a modal action
+ * form.
+ */
+export class ActionFormResponse extends FormResponse {
+    /**
+     * If true, the form was canceled by the player (e.g., they
+     * selected the pop-up X close button).
+     */
+    readonly 'isCanceled': boolean;
+    /**
+     * Returns the index of the button that was pushed.
+     */
+    readonly 'selection': number;
+    protected constructor();
+}
+/**
+ * Base type for a form response.
+ */
+export class FormResponse {
+    /**
+     * If true, the form was canceled by the player (e.g., they
+     * selected the pop-up X close button).
+     */
+    readonly 'isCanceled': boolean;
+    protected constructor();
+}
+/**
+ * Builds a simple two-button modal dialog.
+ */
+export class MessageFormData {
+    /**
+     * @remarks
+     * Method that sets the body text for the modal form.
+     * @param bodyText
+     */
+    body(bodyText: string): MessageFormData;
+    /**
+     * @remarks
+     * Method that sets the text for the first button of the
+     * dialog.
+     * @param text
+     */
+    button1(text: string): MessageFormData;
+    /**
+     * @remarks
+     * This method sets the text for the second button on the
+     * dialog.
+     * @param text
+     */
+    button2(text: string): MessageFormData;
+    /**
+     * @remarks
+     * Creates a new modal form builder.
+     */
+    constructor();
```

Comment:
```suggestion
```
we advice to skip this, the parameterless ctor is always there

Reply:
Made an update to remove these.